eventsourcing/src/main/scala/com/evolutiongaming/akkaeffect/eventsourcing/JournalKeeper.scala

```diff
@@ -71,7 +71,7 @@ object JournalKeeper {
     */
   def of[F[_]: Concurrent: Clock, Sn, St](
     candidate: Candidate[St],
-    snapshotOffer0: Option[SnapshotMetadata],
+    metadata: Option[SnapshotMetadata],
```

Comment:
why not `snapshotMetadata` as in all other cases?

Reply:
there's no other metadata and I prefer keeping names short enough